skietbaan_fe/src/index.js

```diff
@@ -5,14 +5,18 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+//import { Provider } from 'react-redux'
+//import configureStore from './store';
 import { Provider } from 'react-redux'
 import configureStore from './store';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
-ReactDOM.render(<Provider store={configureStore()}>
+
+ReactDOM.render(<App />, document.getElementById('root'));
+/** ReactDOM.render(<Provider store={configureStore()}>
```

Comment:
More GC

Reply:
what do you mean?

Reply:
GC stands for Ghost Code - its commented out code, it should either be removed OR an explanation should be given why it is there

Reply:
explanation in the form of a TODO:

Reply:
Thanks Ash, I have stated my reason for leaving a GC

Reply:
@mthembbu It should be like this:
*/**TODO: Still need to confirm if the App.js browserouter can work without the <Provider>*/*
then here can be whatever commented out code